### DISCUSSION TITLE
Question: Why sudo required on macOS?

### DISCUSSION THREAD

**Discussion by jdolitsky** [rancher-sandbox/rancher-desktop#1381] [Q&A] [answered]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://github.com/rancher-sandbox/rancher-desktop/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

Tried running Rancher Desktop on mac, and noticed that it requires sudo privileges.

Could you explain why this is necessary? Is there some way to prevent it?

Running lima directly does not seem to require this.

### Proposed Solution

N/A

**Comment by evertonlperes** [comment, marked as answer]
Hi @jdolitsky,

The sudo rights is required the following items:
* install vmnet related binaries into /opt/rancher-desktop
* create /etc/sudoers.d/rancher-desktop-lima
* start lima instance using a vmnet instance
* overwrite /var/run/docker.sock
* create symlinks in /usr/local/bin

We're working on this Epic https://github.com/rancher-sandbox/rancher-desktop/issues/1224 in order to improve the sudo process.

**Reply by jdolitsky** [reply]
thanks for clarifying!
